service/application/src/test/java/com/solace/maas/ep/common/model/SempEntityTypeTest.java

```diff
@@ -0,0 +1,28 @@
+package com.solace.maas.ep.common.model;
+
+import org.junit.jupiter.api.Test;
+
+public class SempEntityTypeTest {
```

Comment:
I am not sure if I understand the purpose of this test class, are we testing enum functionality here? 
I think we do not need these tests

Reply:
Agreed